.buildkite/pipeline.yml

```diff
@@ -298,19 +298,19 @@ steps:
 
       # TODO: add/exercise edmf code
       - label: ":computer: Unthreaded performance target (Rosenbrock)"
-        command: "julia --color=yes --project=perf perf/benchmark.jl --job_id perf_target_unthreaded --ode_algo ODE.Rosenbrock23 --enable_threading false --forcing held_suarez --vert_diff true --surface_scheme bulk --moist equil --rad allskywithclear --microphy 0M --dt 1secs --t_end 10secs --dt_save_to_sol Inf --z_elem 25 --h_elem 12"
-        artifact_paths: "perf_target_unthreaded/*"
+        command: "julia --color=yes --project=perf perf/benchmark.jl --job_id perf_target_unthreaded_rosenbrock --ode_algo ODE.Rosenbrock23 --enable_threading false --forcing held_suarez --vert_diff true --surface_scheme bulk --moist equil --rad allskywithclear --microphy 0M --dt 1secs --t_end 10secs --dt_save_to_sol Inf --z_elem 25 --h_elem 12"
+        artifact_paths: "perf_target_unthreaded_Rosenbrock/*"
         agents:
           slurm_mem: 20GB
 
-      - label: ":computer: Unthreaded performance target ARS343"
-        command: "julia --color=yes --project=perf perf/benchmark.jl --job_id perf_target_unthreaded_ARS343 --enable_threading false --forcing held_suarez --vert_diff true --surface_scheme bulk --moist equil --rad allskywithclear --microphy 0M --ode_algo ARS343 --dt 1secs --t_end 10secs --dt_save_to_sol Inf --z_elem 25 --h_elem 12"
-        artifact_paths: "perf_target_unthreaded_ARS343/*"
+      - label: ":computer: Unthreaded performance target"
+        command: "julia --color=yes --project=perf perf/benchmark.jl --job_id perf_target_unthreaded --enable_threading false --forcing held_suarez --vert_diff true --surface_scheme bulk --moist equil --rad allskywithclear --microphy 0M --dt 2secs --t_end 10secs --dt_save_to_sol Inf --z_elem 25 --h_elem 12"
```

Comment:
I don't know if we want to change dt for performance target. @charleskawczynski ?

Reply:
Yeah, no, that's true. I should have not changed the perf ones. I'll change them back